.github/actions/milestone-changelog/action.yml

```diff
@@ -36,6 +36,78 @@ runs:
       with:
         token: ${{ inputs.token }}
         pull_requests: ${{ steps.merged_milestone.outputs.prs }}
+        # section:forced_impact_level
+        allowed_sections: |
```

Comment:
I assume that it would be more convenient for the developers to autogenerate this list to ensure that all modules are present. Anyway, let's merge it as is.